src/workloads/execution/CreateIndex.yml

```diff
@@ -0,0 +1,163 @@
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+
+- Name: IndexCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: Create2dIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            a: "2d"
+            min: -180
+            max: 180
+            bits: 32
+          name: 2dIndex
+    - OperationMetricsName: DropCreated2dIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: Collection0
+        index: 2dIndex
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: Create2dSphereIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key:
+            b: "2dsphere"
+            c: "2dsphere"
+            name: 1
+          name: 2dsphereIndex
+    - OperationMetricsName: DropCreated2dSphereIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: Collection0
+        index: 2dsphereIndex
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: CreateGeoHaystackIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            a: "geoHaystack"
+            d: 1
+          bucketSize: 5
+          name: geoHaystackIndex
+    - OperationMetricsName: DropCreatedGeoHaystackIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: Collection0
+        index: geoHaystackIndex
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: CreateHashedIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            d: "hashed"
+          name: hashedIndex
+    - OperationMetricsName: DropCreatedHashedIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: Collection0
+        index: hashedIndex
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: CreateTextIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            a: "text"
+            b: "text"
+          name: textIndex
+          weights:
+            a: 5
+            b: 10
+    - OperationMetricsName: DropCreatedTextIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: Collection0
+        index: textIndex
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: CreateUniqueIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            d: 1
+            e: 1
```

Comment:
Remove compound index key. Ideally, I don't think we should test "2dsphere" in this workload to keep it simple and have less coverage.